airflow/executors/executor_loader.py

```diff
@@ -25,11 +25,11 @@ class ExecutorLoader:
     Keeps constants for all the currently available executors.
     """
 
-    LOCAL_EXECUTOR = "LocalExecutor"
-    SEQUENTIAL_EXECUTOR = "SequentialExecutor"
-    CELERY_EXECUTOR = "CeleryExecutor"
-    DASK_EXECUTOR = "DaskExecutor"
-    KUBERNETES_EXECUTOR = "KubernetesExecutor"
+    LOCAL_EXECUTOR: str = "LocalExecutor"
+    SEQUENTIAL_EXECUTOR: str = "SequentialExecutor"
+    CELERY_EXECUTOR: str = "CeleryExecutor"
+    DASK_EXECUTOR: str = "DaskExecutor"
+    KUBERNETES_EXECUTOR: str = "KubernetesExecutor"
```

Comment:
Do we need to add `str` here? this are static variables that would always contain string ??

May be yes, may be no.

Just laying out my thought

Reply:
Not really :)